api.md

```diff
@@ -1036,6 +1037,50 @@ CALL distributed_exec('CREATE DATABASE dist_database', transactional => FALSE);
 
 ---
 
+## create_distributed_restore_point() [](create_distributed_restore_point)
+
+Create a named point for performing restore across a distributed database.
+
+This function is an analogue of [pg_create_restore_point](https://www.postgresql.org/docs/current/functions-admin.html) designed
```

Comment:
I still think it is good to put the function name into code as in the rest of the doc:
```suggestion
This function is an analogue of [`pg_create_restore_point`](https://www.postgresql.org/docs/current/functions-admin.html) designed
```

Reply:
You sure we need that change? it doesn't make much sense from markdown perspective, since link will be highlighted anyway. Also couldn't find any similar way of doing as you suggested in the API doc, do you have any reference?

Reply:
It does. See an example in the first sentence under https://docs.timescale.com/latest/api#attach_data_node